ansible_collections/arista/avd/molecule/eos_cli_config_gen_v4.0/inventory/host_vars/traffic-policies.yml

```diff
@@ -25,33 +28,33 @@ traffic_policies:
             prefix_lists:
               - DEMO-01
           protocols:
-            tcp:
+            - name: tcp
```

Comment:
I think this should be `protocol` as the key.

Reply:
okay